docs/guide/conditional-fields.md

```diff
@@ -33,11 +33,70 @@ add: {
 }
 ```
 
+## Complex conditions
+
+In addition to simple field names, the conditional object can take the name of a method as a key. The sibling field value will be compared using strict equivalence to the value returned from the method. Like simple conditional fields, the returned value should be a string, number, or boolean.
+
+This conditional method can either be defined in the `methods` section of the same module as the conditional field, or another module by prefixing the method with the name of defining module followed by a colon. In either case, the method name must have parentheses appended to the end.
+
+::: note
+The property here is a string, not the actual method, so you can't pass arguments back to the method within the parentheses.
+:::
+
+The method will recieve values of `(req, {docId})`, where the `docId` is `null` if the document is being created for the first time. Otherwise, the `docId` will contain the `_id` for a piece/page, or if the method is being called from a widget, the `_id` of the document where the widget is being added. The call is a server-side, asynchronous method, just like that supported for dynamic selection choices. It only occurs when the editor modal is first opened. This is unlike simple conditional fields which continuously poll other schema fields in the same module. Any changes that occur while the editor is open will not be reflected in the visibility of the conditional field. Additionally, the returned value is cached on the first call, so if multiple fields depend on the same method, that method will only be called once.
+
+<AposCodeBlock>
+
+``` javascript
+// A field schema's `add` configuration
+add: {
+  selectSponsors: {
+    label: 'Select a project sponsor',
+    type: 'select',
+    // populate choices dynamically
+    choices: 'sponsorNames',
+    if: {
+      // `()` are mandatory, method defined in the `article` module
+      `isSponsored()`: true
+    }
+  },
+  grantName: {
+    label: 'Select a grant',
+    type: 'select',
+    choices: 'grantNames',
+    if: {
+      // method defined in `modules/grant/index.js`
+      'grant:multipleFundingSources()': 'multiple'
+    }
+  },
+  // additional field options
+methods(self) {
```

Comment:
this me being a little OCDed, but could it be re-indented and moved outside `add`? Maybe writing the whole `fields: { add: {} }, methods()` would be more intuitive for lazy persons like me who like to copy/paste :)